src/commonMain/kotlin/exchange.dydx.abacus/state/v2/supervisor/AccountSupervisor.kt

```diff
@@ -784,28 +784,27 @@ internal open class AccountSupervisor(
 
                 val isStatusValid = status != ComplianceStatus.UNKNOWN
 
-                val body: Map<String, String> =
-                    if (isKeplr == true) {
-                        iMapOf(
-                            "address" to address.rawAddress,
-                            "message" to message,
-                            "action" to complianceAction.toString(),
-                            "signedMessage" to signedMessage!!,
-                            "pubkey" to publicKey!!,
-                        )
-                    } else {
-                        iMapOf(
-                            "address" to address.rawAddress,
-                            "message" to message,
-                            "currentStatus" to status.toString(),
-                            "action" to complianceAction.toString(),
-                            "signedMessage" to signedMessage!!,
-                            "pubkey" to publicKey!!,
-                            "timestamp" to timestamp!!,
-                        )
-                    }
-
                 if (isUrlAndKeysPresent && isKeplrOrHasTimestamp && isStatusValid) {
+                    val body: Map<String, String> =
+                        if (isKeplr == true) {
+                            iMapOf(
+                                "address" to address.rawAddress,
+                                "message" to message,
+                                "action" to complianceAction.toString(),
+                                "signedMessage" to signedMessage!!,
```

Comment:
[nit] we shouldn't need the non-null casting anymore, right?

Reply:
still need to do for the type check haha